sites/gent/macros.tex

```diff
@@ -9,7 +9,8 @@
 \macro{\hpcTeam}{UGent HPC team}
 \macro{\hpcname}{hpcugent}
 \macro{\loginnode}{login.hpc.ugent.be}
-\macro{\loginhost}{gligar01.gligar.gent.vsc}
+\macro{\loginhost}{gligar04.gastly.os}
+\macro{\altloginhost}{gligar05.gastly.os}
```

Comment:
why use the OS network? Use the VSC names of the machines? Those work from everywhere within the VSC network.

Reply:
Because this corresponds to what they see when they run `hostname` or `hostname -f`